### PR TITLE
[3460][IMP] sale_tier_validation: hides action_quotation_send before record validated

### PR DESCRIPTION
**Pull request by rinaldifirdaus**
[3460](https://www.quartile.co/web#view_type=form&model=project.task&id=3460&active_id=3460&menu_id=)

### CODE REVIEW

File: sale_tier_validation/views/sale_order_view.xml
Comment (kanda999) [inline sale_tier_validation/views/sale_order_view.xml]: Is `('validated','=',False),('need_validation','=',False)` necessary?
Is it covered only by `('validated','=',False)`?

**Review by kanda999 — CHANGES_REQUESTED** [review]

File: sale_tier_validation/views/sale_order_view.xml
Comment (rinaldifirdaus) [reply sale_tier_validation/views/sale_order_view.xml]: hmm, i think so. i was not able to use the validated = false because i didnt add it to view as invisible. let me try and check again. thanks Kanda san

Comment (rinaldifirdaus) [reply sale_tier_validation/views/sale_order_view.xml]: Just tested on my local, it work only by ('validated','=',False). Thank you

Comment (yostashiro) [reply sale_tier_validation/views/sale_order_view.xml]: @rinaldifirdaus What about the case where validation is not required in the first place (according to the tier setting)? Not sure such case exists at the client, but we should consider this case if we are to update the OCA module.

**Comment by rinaldifirdaus** [comment]
on the new commit, currently the send email button will not be displayed before request validation sent by the users to Approvers.
for the record that has been approved and the records that doesn't need approval, the button will be displayed.
the button will also be displayed after users click request validation before approved even its not approved yet, but the user will not be allowed to send the quotation before its validated.

File: sale_tier_validation/views/sale_order_view.xml
Comment (yostashiro) [inline sale_tier_validation/views/sale_order_view.xml]: Put this record on top.  We usually have other views above search views.

File: sale_tier_validation/views/sale_order_view.xml
Comment (yostashiro) [inline sale_tier_validation/views/sale_order_view.xml]:
```suggestion
                <field name="need_validation" invisible="1" />
                <field name="next_review" invisible="1" />
```

File: sale_tier_validation/views/sale_order_view.xml
Comment (yostashiro) [inline sale_tier_validation/views/sale_order_view.xml]:
```suggestion
                <attribute name="attrs">{'invisible':['|','|',('state','!=','draft'),('need_validation','=',True),('next_review','!=','')]}</attribute>
```

File: sale_tier_validation/views/sale_order_view.xml
Comment (yostashiro) [inline sale_tier_validation/views/sale_order_view.xml]:
```suggestion
```

**Review by kanda999 — APPROVED** [review]
LGTM 👍

**Pull request merged**
